3.10/analyzers.md

```diff
@@ -849,8 +922,8 @@ An Analyzer capable of removing specified tokens from the input.
 
 It uses binary comparison to determine if an input token should be discarded.
 It checks for exact matches. If the input contains only a substring that
-matches one of the defined stopwords, then it is not discarded. Longer inputs
-such as prefixes of stopwords are also not discarded.
+matches one of the defined stop words, then it is not discarded. Longer inputs
```

Comment:
I'd say "stopwords" should be used as it matches what we have in the options. No strong opinion though.

Reply:
The correct spelling appears to be "stop words", which is what we should use if you refer to the concept, and `stopwords` if it's a code value because that's how the API is.